CheckUserInput/CheckUserInput.Core/P2funcs.cs

```diff
@@ -1,19 +0,0 @@
-﻿using System;
-
-namespace CheckUserInput.Core
```

Comment:
As I see this Core module doesn't have any logic anymore (as P2funcs was the only class inside) do you plan to add anything there? If yes, please do so, if no - please remove it. In general you don't want to keep an empty module :)

Reply:
Hm not sure how this is here since i deleted it, anyways I made sure with last commit. Fixed.